lib/ui/compositing.dart

```diff
@@ -47,7 +47,8 @@ class Scene extends NativeFieldWrapperClass1 {
     if (width <= 0 || height <= 0) {
       throw Exception('Invalid image dimensions.');
     }
-    return _futurize((_Callback<Image?> callback) => _toImage(width, height, (_Image? image) {
+    final _Image image = _Image._();
+    return _futurize((_Callback<Image?> callback) => _toImage(width, height, image, (_Image? image) {
```

Comment:
We should just make this a voidcallback, right?

We're passing in an image and then passing the same handle back it seems.